_posts/2019-04-05-omakase-typescript.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: epic
+title: "What is TypeScript?"
+date: "2019-05-05"
+author: [orta]
+categories: [community, javascript, roads and bridges, typescript]
+---
+
+TypeScript is a language which builds on JavaScript. This post is a non-technical overview of what JavaScript is,
+how TypeScript extends it and why we choose to adopt it at Artsy.
+
+<!-- more -->
+
+## What is JavaScript?
+
+First up, you can't describe TypeScript without talking about JavaScript. To create a website (and a bunch of other
+types of things) you work in three languages: HTML, CSS and JavaScript (JS). The first two languages about
+presenting the look and feel of the website, and JavaScript allows you to edit what it shows and make it
+interactive.
+
+We describe having these sets of skills as being a "front-end" developer. You have to understand those three
+languages to present anything inside a web browser like Safari, Firefox or Chrome. So, given how popular the web
+is, there is a massive demand for people who are good at using these three languages.
+
+There is also the set of skills for the "back-end" developers, which are to create computer servers which
+communicate either to a web browser (by passing it HTML/CSS/JS) or to another computer server (by sending a kind of
+JavaScript.) You don't need to use HTML, CSS or JS to write this type of code, but it's usually an end-product of
+your work. We mostly build our back-ends in Ruby or JavaScript at Artsy.
+
+### What do Programming Languages do?
+
+Programming languages are an interesting problem to solve. People read code many, many multiples of times more than
+they write it - so developers create languages which are good at solving particular problems in a terse amount of
```

Comment:
Hrm, I do want to use "terse" here over "small" because that comes with extra context of doing the same thing but with less. Will figure out how people use the word properly

Reply:
small works fine in this context 👍